src/compiletest/runtest.rs

```diff
@@ -927,6 +927,13 @@ fn check_expected_errors(expected_errors: Vec<errors::ExpectedError>,
         format!("{}:{}:", testfile.display(), ee.line)
     }).collect::<Vec<String>>();
 
+    let (expect_help, expect_note) =
+        expected_errors.iter()
+                        .fold((false, false),
+                              |(acc_help, acc_note), ee|
+                                  (acc_help || ee.kind == "help:", acc_note ||
```

Comment:
Yep, thanks for making that clear. I probably should have been more specific in the description. By the way, is there a place to document this?